controllers/weightsandbiases_controller.go

```diff
@@ -46,6 +47,7 @@ import (
 )
 
 const resFinalizer = "finalizer.app.wandb.com"
+const licenseSecretName = "wandb-license-secret" // Constant for the secret name
```

Comment:
This need to handle the scenario where the user provides the license via a user created secret https://github.com/wandb/helm-charts/blob/main/charts/operator-wandb/values.yaml#L18